test/test_jsinterp.py

```diff
@@ -5,7 +5,10 @@
 # Allow direct execution
 import os
 import sys
-import unittest
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
That's fine. Don't forget to add the dependency in requirements.txt and install it in .travis.yml. See https://github.com/rg3/youtube-dl/pull/7852/files